### PR TITLE
TextComponent is now Text. Add shared colors css obj.

### PR DESCRIPTION
**Pull request by patrickkim**

### CODE REVIEW

File: src/atoms/Text/index.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+   * provided in the component's index.js file.
+   */
+  className: PropTypes.string,
+  /**
+   * You can use any html element text tag
+   */
+  tag: PropTypes.string,
+
+  /**
+   * determines typography class
```
Comment (drewdrewthis) [inline src/atoms/Text/index.js]: Capital 'D'?

File: src/atoms/Text/index.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+   */
+  type: PropTypes.number,
+
+  /**
+   * Font weight: light|regular|bold
+   * - light = 300
+   * - semibold = 500
+   * - bold = 600
```
Comment (drewdrewthis) [inline src/atoms/Text/index.js]: This is really confusing. My understanding is that we only have light, regular, and semibold options. `regular` should probably be the default.

File: src/atoms/TextComponent/Readme.md [hunk 3]
```diff
@@ -1,14 +1 @@
...
-    <TextComponent type={1}>Type 1</TextComponent>
-    <TextComponent type={2}>Type 2</TextComponent>
-    <TextComponent type={3}>Type 3</TextComponent>
-    <TextComponent type={4}>Type 4</TextComponent>
-    <TextComponent type={5}>Type 5</TextComponent>
-    <TextComponent type={6}>Type 6</TextComponent>
-    <TextComponent type={7}>Type 7</TextComponent>
-  </div>
-```
+To be deprecated. Use `<Text />`
```
Comment (drewdrewthis) [inline src/atoms/TextComponent/Readme.md]: :+1:

File: src/atoms/Text/__tests__/text.spec.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import { shallow, mount } from 'enzyme';
+
+import Text from 'atoms/Text';
+
+describe('<Text />', () => {
+
+  it('renders', () => {
+    const wrapper = shallow(<Text />);
+
+    expect(wrapper.length).to.equal(1);
```
Comment (drewdrewthis) [inline src/atoms/Text/__tests__/text.spec.js]: Please change this to the `expect(wrapper.type()).to.equal(Text);` that we'd discussed.

File: src/atoms/TextComponent/index.js [hunk 5]
```diff
@@ -1,92 +1,4 @@
...
-};
-
-
-TextComponent.defaultProps = {
-  tag: 'p',
-  type: 6
-};
-
-export default TextComponent;
+export default props => <Text {...props} />;
```
Comment (drewdrewthis) [inline src/atoms/TextComponent/index.js]: Slick!

Comment (drewdrewthis) [inline src/atoms/TextComponent/index.js]: Can you not just `export default Text`?

File: src/organisms/cards/RadioCard/__tests__/radio_card.spec.js [hunk 6]
```diff
@@ -7,7 +7,7 @@ import RadioCard from 'organisms/cards/RadioCard';
 describe('<RadioCard />', () => {
 
   it('renders', () => {
-    const wrapper = shallow(<RadioCard label='its a label' input={{}} />);
+    const wrapper = shallow(<RadioCard label='its a label' input={{}} radioValue='something' />);
 
     expect(wrapper.length).to.equal(1);
```
Comment (drewdrewthis) [inline src/organisms/cards/RadioCard/__tests__/radio_card.spec.js]: Same comment as above.

File: src/atoms/Text/index.js (hunk 2 above)
Comment (patrickkim) [inline src/atoms/Text/index.js]: It's confusing because the doc was wrong.

File: src/atoms/Text/__tests__/text.spec.js (hunk 4 above)
Comment (patrickkim) [inline src/atoms/Text/__tests__/text.spec.js]: `type` doesn't work like that. In most cases type will be `<div>`, the wrapping element.  If you had a component that returns `<Something>thing</Something>` its type will be `Something`.

File: src/atoms/TextComponent/index.js (hunk 5 above)
Comment (patrickkim) [inline src/atoms/TextComponent/index.js]: Either the linter was failing or it didn't work, I forget which didn't give it much thought.

**Pull request merged**